dist/index.js

```diff
@@ -96,15 +96,14 @@ function generateIgnore(ig, codeOwnerPath) {
 exports.generateIgnore = generateIgnore;
 function getChangedFiles(client, prNumber, includeDeleted) {
     return __awaiter(this, void 0, void 0, function* () {
-        const listFilesOptions = client.rest.pulls.listFiles.endpoint.merge({
+        const listFilesResponse = yield client.paginate(client.rest.pulls.listFiles, {
             owner: github.context.repo.owner,
             repo: github.context.repo.repo,
             pull_number: prNumber,
         });
-        const listFilesResponse = yield client.paginate(listFilesOptions);
         const changedFiles = listFilesResponse
-            .filter((f) => includeDeleted || f.status !== 'deleted')
-            .map((f) => f.filename);
+            .filter((file) => includeDeleted || file.status !== 'removed')
```

Comment:
🙏

Reply:
Yeah, I noticed I'd written the wrong status so it was never actually removing deleted files. 😐